CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 15.1.3 (2024-10-09)
+### Features
+* **o-chart**:
+  * Added auto-scaling to the axis of the line chart ([0abc142]https://github.com/OntimizeWeb/ontimize-web-ngx-charts/commit/0abc142) Closes [#264](https://github.com/OntimizeWeb/ontimize-web-ngx-charts/issues/264)
```

Comment:
```suggestion
  * Added auto-scaling to the axis of the line chart ([0abc142](https://github.com/OntimizeWeb/ontimize-web-ngx-charts/commit/0abc142)) Closes [#264](https://github.com/OntimizeWeb/ontimize-web-ngx-charts/issues/264)
```
The links to commit is wrong
![image](https://github.com/user-attachments/assets/bb5543e4-5afa-44a3-b169-82236517ee4d)